schema/type.schema.json

```diff
@@ -101,154 +82,116 @@
-            "minItems": 1,
-            "maxItems": 3,
-            "uniqueItems": true
-        },
-        "canonSpec": {
-            "type": "object",
-            "properties": {
-                "OT": {
-                    "type": "object",
-                    "properties": {
-                        "books": {"items": {"$ref": "#/definitions/bookOT"}}
-                    },
-                    "oneOf": [
-                        {"$ref": "#/definitions/canonComponentOT"},
-                        {"$ref": "#/definitions/canonComponentCustom"}
-                    ],
-                    "required": ["name"]
-                },
-                "OT+": {
-                    "type": "object",
-                    "properties": {
-                        "books": {"items": {"$ref": "#/definitions/bookOTDC"}}
-                    },
-                    "oneOf": [
-                        {"$ref": "#/definitions/canonComponentOT+"},
-                        {"$ref": "#/definitions/canonComponentCustom"}
-                    ],
-                    "required": ["name"]
-                },
-                "DC": {
-                    "type": "object",
-                    "properties": {
-                        "books": {"items": {"$ref": "#/definitions/bookDC"}}
-                    },
-                    "oneOf": [
-                        {"$ref": "#/definitions/canonComponentDC"},
-                        {"$ref": "#/definitions/canonComponentCustom"}
-                    ],
-                    "required": ["name"]
-                },
-                "NT": {
-                    "type": "object",
-                    "properties": {
-                        "books": {"items": {"$ref": "#/definitions/bookNT"}}
-                    },
-                    "oneOf": [
-                        {"$ref": "#/definitions/canonComponentNT"},
-                        {"$ref": "#/definitions/canonComponentCustom"}
-                    ],
-                    "required": ["name"]
-                }
-            },
-            "additionalProperties": false,
-            "minProperties": 1,
-            "allOf": [
-                {"not": {"required": ["OT", "OT+"]}},
-                {"not": {"required": ["DC", "OT+"]}}
-            ],
-            "$comment": "OT or DC may not be specified if OT+ is present."
-        }
+	    "properties": {
+		"name": {
+		    "type": "string",
+		    "enum": ["scripture", "gloss", "parascriptural", "peripheral"]
+		},
+		"flavor": {
+		    "type": "object",
+		    "oneOf": [
+			{
+			    "properties": {
+				"name": {
+				    "type": "string",
+				    "pattern": "^x-[a-z][A-za-z0-9]*$"
+				}
+			    }
+			},
+			{
+			    "properties": {
+				"name": {
+				    "type": "string",
+				    "enum": [
+					"TextTranslation",
+					"AudioTranslation",
+					"SignLanguageTranslation",
+					"BrailleScripturePublication",
+					"TextStories",
+					"WordAlignment",
+					"Versification"
```

Comment:
Let's make this lowerCamelCase.  The notion that this is "kind of like a class" isn't an obvious reason to break consistency to me.